src/ApiService/ApiService/onefuzzlib/Containers.cs

```diff
@@ -255,4 +281,46 @@ public class Containers : IContainers {
         }
         return OneFuzzResultVoid.Ok;
     }
+
+    public async Async.Task DeleteAllExpiredBlobs() {
+        var storageTypes = new List<StorageType> { StorageType.Corpus, StorageType.Config };
+        var allStorageAccounts = storageTypes.Select(_context.Storage.GetAccounts)
+            .SelectMany(x => x);
+
+        foreach (var storageAccount in allStorageAccounts) {
+            await DeleteExpiredBlobsForAccount(storageAccount);
```

Comment:
Let's do deletions for each storage account in parallel; they won't interfere with each other.